src/mem/epoch/garbage.rs

```diff
@@ -31,17 +31,20 @@ impl Bag {
         Bag(vec![])
     }
 
-    fn insert<T>(&mut self, elem: *mut T) {
+    fn insert<T>(&mut self, elem: *mut T, do_drop: bool) {
```

Comment:
Instead of having flag arguments like this you could have separate functions. For example, `insert` and `insert_and_drop`. What do you think about that?